scripts/check-public-api.py

```
@@ -109,6 +110,225 @@ def render_public_api(repo: Path, feature_args: list[str]) -> str:
+        str(repo / "target"),
+        "--lib",
+        "--locked",
+        *feature_args,
+        "--",
+        "-Z",
+        "unstable-options",
+        "--output-format",
+        "json",
+    ]
+    try:
+        subprocess.run(command, check=True, text=True, capture_output=True, cwd=repo)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)